components/director/internal/model/entitytype.go

```diff
@@ -0,0 +1,168 @@
+package model
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/pagination"
+	"github.com/kyma-incubator/compass/components/director/pkg/resource"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/str"
+)
+
+// EntityType missing godoc
+type EntityType struct {
+	ApplicationID                *string
+	ApplicationTemplateVersionID *string
+	OrdID                        string
+	LocalID                      string
+	CorrelationIDs               json.RawMessage
+	Level                        string
+	Title                        string
+	ShortDescription             *string
+	Description                  *string
+	SystemInstanceAware          *bool
+	ChangeLogEntries             json.RawMessage
+	PackageID                    string
+	Visibility                   string
+	Links                        json.RawMessage
+	PartOfProducts               json.RawMessage
+	PolicyLevel                  *string
+	CustomPolicyLevel            *string
+	ReleaseStatus                string
+	SunsetDate                   *string
+	Successors                   json.RawMessage
+	Extensible                   json.RawMessage
+	Tags                         json.RawMessage
+	Labels                       json.RawMessage
+	DocumentationLabels          json.RawMessage
+	ResourceHash                 *string
+	Version                      *Version
+	*BaseEntity
+}
```

Comment:
`lastUpdate` field is missing. By the ORD spec it is part of the Entity Type resource. - https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document/#entity-type_lastupdate